pyproject.toml

```diff
@@ -0,0 +1,76 @@
+[build-system]
+requires = ["scikit-build-core", "numpy"]
+build-backend = "scikit_build_core.build"
+
+[project]
+name = "spglib"
+version = "2.1.0-rc1"
```

Comment:
Can you add a description at `Contribution.md` that we also need to bump up a version at `pyproject.toml`?